.rubocop.yml

```diff
@@ -93,3 +94,14 @@ Style/SafeNavigation:
 
 Style/RegexpLiteral:
   Enabled: false
+
+Security/Open:
+  Exclude:
+    - 'app/services/getty_parser.rb'
```

Comment:
Checks for `Kernel/open` but I don't see it, do you??

Reply:
Yeah. https://github.com/pulibrary/cicognara-rails/blob/f4ccc90e5359d3bf69f7dba46af119af4a0204bd/app/services/getty_parser.rb#L56